src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js

```diff
@@ -669,6 +670,38 @@ describe('Testing Edit concept actions ', () => {
     });
   });
 
+  it('should unretire a Mapping when the unretireMapping action is triggered', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({ status: 200, response: existingConcept });
+    });
+    const expectedActions = [
+      { type: IS_FETCHING, payload: true },
+      { type: UPDATE_CONCEPT, payload: existingConcept },
+    ];
+    const store = mockStore(mockConceptStore);
+    const conceptUrl = '/orgs/EthiopiaNHDD/sources/HMIS-Indicators/concepts/C1.1.1.1/';
+    return store.dispatch(unretireMapping(conceptUrl)).then(() => {
+      expect(store.getActions()).toEqual(expectedActions);
```

Comment:
How does this assert that a mapping is now unretired?

Reply:
I have made the test more specific:
`expect(result.retired).toEqual(false);`